src/lexer.rs

```diff
@@ -12,7 +12,6 @@ use crate::FunctionIdentifier::{Cbrt, Ceil, Cos, Exp, Abs, Floor, Ln, Log, Round
 use crate::units::Unit;
 use crate::units::Unit::*;
 use unicode_segmentation::{Graphemes, UnicodeSegmentation};
-use std::cmp::Ordering;
```

Comment:
It would be cleaner to just squash this with the original (relevant) commit to avoid cluttering the git history unnecessarily.